cryptostore/data/storage.py

```diff
@@ -12,17 +12,17 @@
 
 
 class Storage(Store):
-    def __init__(self, config):
+    def __init__(self, config, parquet_buffer):
```

Comment:
see previous comment. better to have kwargs and have the parquet specific implementation read that from out of it

Reply:
@bmoscon : done